src/bootloader_env/grub.rs

```diff
@@ -8,7 +8,7 @@ pub fn bootloader_env(key: &str) -> Result<String> {
         .arg(GRUB_ENV_FILE)
         .arg("list")
         .output()
-        .with_context(|| "failed to execute 'grub-editenv list'")?;
+        .context("failed to execute 'grub-editenv list'")?;
```

Comment:
why not doing that lazy? what is the agenda?

Reply:
why do it lazy? Do you see heavy work somewhere that is handled in `context`? I searched for a recommendation with the result that you should use `with_context` if you have heavy work to do in a closure. If you intend to just add a string `context` is the choice